var/spack/repos/builtin/packages/fftw/package.py

```diff
@@ -33,7 +33,7 @@ class Fftw(AutotoolsPackage):
     patch('pfft-3.3.5.patch', when="@3.3.5:+pfft_patches", level=0)
     patch('pfft-3.3.4.patch', when="@3.3.4+pfft_patches", level=0)
     patch('pgi-3.3.6-pl2.patch', when="@3.3.6-pl2%pgi", level=0)
-    patch('intel-configure.patch', when="@3:3.3.8%intel", level=0)
+    patch('a64fx-3.3.8.patch', when="@3.3.8 target=a64fx")
```

Comment:
Can you provide a reference where this patch is discussed? Asking since the patch changes ~8k lines in the sources. Will it be accepted upstream for future releases?

Reply:
(See https://github.com/spack/spack/pull/18349#discussion_r480033722 for the initial discussion on the question above)